### PR TITLE
Removed legacy Cloud infrastructure info

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request
Update Cloud documentation to remove legacy infrastructure references and instructions:

- [x] Remove references to projects provisioned before October 23, 2017
- [x] Remove "submit support ticket" references for environment config changes on Pro Staging and Production environments
- [x] Remove [Pro Architecture (legacy)](https://devdocs.magento.com/guides/v2.3/cloud/architecture/pro-architecture-legacy.html) topic
- [x] Remove instructions for migrating projects from legacy infrastructure in [Add Staging and Production to Pro Project UI](https://devdocs.magento.com/guides/v2.3/cloud/architecture/pro-architecture-legacy.html) topic
- [x] Remove references to deleted topics, and add redirects as needed
- [x] Clean up markdown syntax

## Validation

- [x] Build preview 
- [ ] Verify that instructions and references to legacy Cloud projects and project configurations have been removed
- [ ] Editorial review

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.2/cloud/architecture/cloud-architecture.md
- https://devdocs.magento.com/guides/v2.2/cloud/architecture/pro-architecture.md
- https://devdocs.magento.com/guides/v2.2/cloud/architecture/pro-develop-deploy-workflow.md
- https://devdocs.magento.com/guides/v2.2/cloud/configure/setup-cron-jobs.md
- https://devdocs.magento.com/guides/v2.2/cloud/env/environments.md
- https://devdocs.magento.com/guides/v2.2/cloud/integrations/bitbucket-integration.md
- https://devdocs.magento.com/guides/v2.2/cloud/live/stage-prod-migrate-prereq.md
- https://devdocs.magento.com/guides/v2.2/cloud/live/stage-prod-migrate.md
- https://devdocs.magento.com/guides/v2.2/cloud/project/magento-env-yaml.md
- https://devdocs.magento.com/guides/v2.2/cloud/project/project-conf-files_magento-app.md
- https://devdocs.magento.com/guides/v2.2/cloud/project/project-conf-files_services-redis.md
- https://devdocs.magento.com/guides/v2.2/cloud/project/project-integrate-blackfire.md
- https://devdocs.magento.com/guides/v2.2/cloud/project/project-multi-sites.md
- https://devdocs.magento.com/guides/v2.2/cloud/project/project-upgrade.md
- https://devdocs.magento.com/guides/v2.2/cloud/project/project-webint-basic.md
- https://devdocs.magento.com/guides/v2.2/cloud/project/project-webint-branch.md
- https://devdocs.magento.com/guides/v2.2/cloud/project/projects.md
- https://devdocs.magento.com/guides/v2.2/cloud/project/user-admin.md
- https://devdocs.magento.com/guides/v2.2/cloud/reference/discover-deploy.md
- https://devdocs.magento.com/guides/v2.2/cloud/release-notes/CloudReleaseNotes2.2.1.md
- https://devdocs.magento.com/guides/v2.2/cloud/release-notes/CloudReleaseNotes2.2.md
- https://devdocs.magento.com/guides/v2.2/cloud/release-notes/cloud-tools.md
- https://devdocs.magento.com/guides/v2.2/cloud/trouble/pro-env-management.md
- https://devdocs.magento.com/guides/v2.2/cloud/trouble/site-availability.md
- https://devdocs.magento.com/guides/v2.2/cloud/trouble/trouble.md
- https://devdocs.magento.com/guides/v2.3/cloud/project/projects.md
- https://devdocs.magento.com/guides/v2.3/cloud/project/user-admin.md
- https://devdocs.magento.com/guides/v2.3/cloud/release-notes/cloud-tools.md

whatsnew
Updated Magento Commerce Cloud documentation to remove obsolete instructions for configuring  legacy environments and migrating legacy Cloud environments to use the Project Web UI. Legacy environments are those provisioned prior to October 23, 2017.

### CODE REVIEW

File: guides/v2.2/cloud/project/project-webint-branch.md [hunk 1]
```diff
@@ -14,21 +14,19 @@ For more information about managing environments using the CLI, see [Manage bran
...
-{% include cloud/wings-management.md %}
-
 ## Add or delete an environment {#project-branch-add}
 
 Complete development of code and added extensions in a branch and, when complete, merge (`git push`) the branch with its parent or master.
 
-* For Starter, we recommend you create a `staging` branch from Master, then branch from `staging` for development.
-* For Pro, you want to branch from Development (Integration `master`).
+- For Starter, we recommend you create a `staging` branch from Master, then branch from `staging` for development.
+- For Pro, you want to branch from the development branch (Integration `master`).
```
Comment (hguthrie) [inline guides/v2.2/cloud/project/project-webint-branch.md]:
```suggestion
- For Pro, create a development branch from the Integration environment.
```

File: guides/v2.2/cloud/project/user-admin.md [hunk 2]
```diff
@@ -57,18 +51,20 @@ You can use the {{site.data.var.ece}} command line client to fully manage your u
...
-* `magento-cloud user:list [users]` lists project users
-* `magento-cloud user:role` views or change a user's role
+- `magento-cloud user:add` adds a user to the project
+- `magento-cloud user:delete` deletes a user
+- `magento-cloud user:list [users]` lists project users
+- `magento-cloud user:role` views or change a user's role
 
 For example, the following command adds the project administrator (`admin`) role to `alice@example.com` and gives her `contributor` privileges to the `development` environment:
 
 The following prompts are displayed:
```
Comment (hguthrie) [inline guides/v2.2/cloud/project/user-admin.md]:
```suggestion
```

File: guides/v2.2/cloud/release-notes/CloudReleaseNotes2.2.1.md [hunk 3]
```diff
@@ -20,9 +20,9 @@ We strongly recommend upgrading to Fastly v1.2.33 module for all Magento Commerc
...
+* **30-day free trials available for Starter and Pro projects**. Starting October 23, 2017, we provide trials for Start and Pro projects. We auto provision your project and configure basic settings to get you started. You can access a new Onboarding Portal to get started with your project. For more information, see [Subscriptions and plans]({{ site.baseurl }}/guides/v2.2/cloud/bk-cloud.html).
 
-* **Staging and Production environments in the UI for Pro projects**. Starting October 23, 2017, all projects created using the free 30-day trial or provisioned for new accounts includes Staging and Production environments in the Project Web Interface. Existing accounts created before October 23, 2017 can enter a ticket to be updated. For more information, see [Add Staging and Production to Pro projects UI]({{ site.baseurl }}/guides/v2.2/cloud/trouble/pro-env-management.html).
+* **Staging and Production environments in the UI for Pro projects**. Starting October 23, 2017, all projects created using the free 30-day trial or provisioned for new accounts includes Staging and Production environments in the Project Web Interface. See
```
Comment (hguthrie) [inline guides/v2.2/cloud/release-notes/CloudReleaseNotes2.2.1.md]: See is left open-ended...

File: guides/v2.3/cloud/project/projects.md [hunk 4]
```diff
@@ -110,7 +106,7 @@ If you have inactive Git branches of code, you can toggle displaying the branche
 
 ## Configure environments {#configure}
 
-You can manage variables and settings for Production, Staging, and Integration environments through this interface, or with CLI commands. Click **Configure environment** to create and manage [*environments*]({{ page.baseurl }}/cloud/env/environments.html), each of which corresponds to a Git branch.
+You can manage variables and settings for Production, Staging, and Integration environments through this interface, or with CLI commands. Click -*Configure environment** to create and manage [*environments*]({{ page.baseurl }}/cloud/env/environments.html), each of which corresponds to a Git branch.
```
Comment (hguthrie) [inline guides/v2.3/cloud/project/projects.md]:
```suggestion
You can manage variables and settings for Production, Staging, and Integration environments through this interface, or with CLI commands. Click **Configure environment** to create and manage [*environments*]({{ page.baseurl }}/cloud/env/environments.html), each of which corresponds to a Git branch.
```

File: guides/v2.2/cloud/release-notes/CloudReleaseNotes2.2.1.md (hunk 3 above)
Comment (meker12) [inline guides/v2.2/cloud/release-notes/CloudReleaseNotes2.2.1.md]:
```suggestion
* **Staging and Production environments in the UI for Pro projects**. Starting October 23, 2017, all projects created using the free 30-day trial or provisioned for new accounts includes Staging and Production environments in the Project Web Interface.
```

File: _includes/cloud/hooks.md [hunk 5]
```diff
@@ -1,4 +1,3 @@
-For technical reasons, you must enter a Support ticket to add your [`.magento.app.yaml`]({{ page.baseurl }}/cloud/project/project-conf-files_magento-app.html) file to Staging and Production environments. This affects all Pro projects created before and after October 23, 2017.
```
[the remark this answers is not in the document]
Comment (meker12) [reply _includes/cloud/hooks.md]: I wasn't sure about the blank line after deleting the first paragraph, so I left it in.   Based on the preview,  the first blank line was ignored.  However, I agree that stray blank lines should go.

File: guides/v2.2/cloud/live/stage-prod-migrate-prereq.md [hunk 6]
```diff
@@ -34,115 +35,104 @@ To prepare your environments for full deployment, you need:
...
+**How it works!** After you configure the agent and settings, you can migrate files between servers using SSH and the `scp` (secure copy) command. The SSH agent forwards authentication requests from Staging or Production environments to your local with a working Magento system, so you can connect using your local private SSH key. After you use Git to push code, you can use SSH to connect to the Staging and Production environments to update code, data, and files.
 
 You can use any terminal client you prefer for SSH access, or see our [Recommended tools]({{ page.baseurl }}/cloud/before/before-workspace.html#recommended-tools). For these examples, we use the OpenSSH client.
 
 To set up an SSH agent:
 
-1.	In a terminal client, log in to your local system.
-2.	Enter the following command to check if the SSH agent is running and list fingerprints of all identities currently represented by the agent:
+1. In a terminal client, log in to your local system.
```
[the remark this answers is not in the document]
Comment (meker12) [reply guides/v2.2/cloud/live/stage-prod-migrate-prereq.md]: Double-checking on this.  I believe that we can remove this and point to the [Add SSH key to your Magento account](https://devdocs.magedevteam.com/720/guides/v2.2/cloud/before/before-workspace-ssh.html#ssh-add-to-account) or maybe to the top-level [Enable SSH keys](https://devdocs.magedevteam.com/720/guides/v2.2/cloud/before/before-workspace-ssh.html).

[the remark this answers is not in the document]
Comment (meker12) [reply guides/v2.2/cloud/live/stage-prod-migrate-prereq.md]: Added inline procedure to add public ssh keys to project.

File: _data/toc/cloud-guide.yml [hunk 7]
```diff
@@ -23,12 +23,6 @@ pages:
         - label: Pro develop and deploy workflow
           url: /cloud/architecture/pro-develop-deploy-workflow.html
 
-        - label: Pro architecture (legacy)
-          url: /cloud/architecture/pro-architecture-legacy.html
-
-        - label: Add Staging and Production to Pro projects UI
-          url: /cloud/trouble/pro-env-management.html
```
[the remark this answers is not in the document]
Comment (meker12) [reply _data/toc/cloud-guide.yml]: 👍 Added a redirect to the [Pro develop and deploy workflow topic](https://devdocs.magento.com/cloud/architecture/pro-develop-deploy-workflow.html).

**Comment by meker12** [comment]
running tests

**Pull request merged**